### PR TITLE
Add min and max to analytics

### PR DESCRIPTION
**Pull request by Robdel12**
# What is this?

This computes the slowest and busiest days for the selected date range on the analytics page. 
# Picture

screencapture-localhost-4200-analytics
## Todos
- [ ] Write tests

### CODE REVIEW

File: ember/app/templates/analytics.hbs [hunk 1]
```diff
@@ -1,11 +1,10 @@
-<form {{action "updateData" on='submit'}}>
-  From: {{date-picker date=graphStartDate format="MMMM Do YYYY" valueFormat='YYYY-MM-DD' maxDate=maxDate placeholder="Start Date"}}
-  To: {{date-picker date=graphEndDate format="MMMM Do YYYY" valueFormat='YYYY-MM-DD' maxDate=maxDate placeholder="End Date"}}
-  <button type="submit" class="btn">Save</button>
+<form>
+  From: {{date-picker date=graphStartDate format="MMMM Do YYYY" valueFormat='YYYY-MM-DD' maxDate=pickerMaxDate placeholder="Start Date"}}
+  To: {{date-picker date=graphEndDate format="MMMM Do YYYY" valueFormat='YYYY-MM-DD' maxDate=pickerMaxDate placeholder="End Date"}}
 </form>
```
Comment (Robdel12) [inline ember/app/templates/analytics.hbs]: An empty form is a pointless one :P

**Comment by tracehelms** [comment]
Are you tracking these analytics in Rails or are you pulling from Google Analytics or something?

**Comment by Robdel12** [comment]
@tracehelms I'm pulling it from Google Analytics.

**Comment by tracehelms** [comment]
Ok, thought so. Cool.

**Pull request closed without merging**